### ISSUE TITLE
minor graphiql UI improvements

### ISSUE THREAD

**Issue by dsandip** [hasura/graphql-engine#1592]
1. From vedic on Discord:
```
in the hasura graphql API query console window , can we have provision to hide the request headers as it is occupying half of the total window
```
2. query window height vs. variables window height

3. @rikinsk

**Comment by rikinsk** [comment]
related to #1624

**Comment by mnlbox** [comment]
@dsandip Thanks for this issue. I'm using `Altair GraphQL Client` until now and read this issue because I think Hasura didn't support query variable. I think it's because of query window height that you are mentioned in number 2 :wink: 
it's a bad UX experience and we need a fix for this point.

**Comment by rikinsk** [comment]
Fixed in #1723
